apps/data_handler/README.md

```diff
@@ -95,6 +95,31 @@ Go to bash
 docker-compose exec backend bash
 ```
 
+
+## Running Migration Command:
```

Comment:
Please fix that, look as it's in master branch
https://github.com/CarmineOptions/derisk-research/blob/master/apps/data_handler/README.md